src/Event/EventTest/DALTests/EventSeatRepositoryTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using DAL;
+using DAL.Events;
+using DAL.Events.Repository;
+using Microsoft.Extensions.DependencyInjection;
+using TestsCore;
+
+namespace EventTest.DALTests
+{
+	public class EventSeatRepositoryTests
+	{
+		[Fact]
+		public async Task GetEventSectionSeats_ExistingValues_EmptyIfInputIncorrect()
+		{
+			var serviceProvider = ServiceConfigurationProvider.Get<EventContext>(services => services.AddEventsRepositories());
```

Comment:
we could make serviceProvider a private readonly field and initialize it in test class constructor. This would simplify the code

Reply:
Same for repository

Reply:
It does, looks great